test/XRP/issued-currency-client.test.ts

```diff
@@ -98,6 +117,39 @@ describe('Issued Currency Client', function (): void {
     })
   })
 
+  it('getTrustLines - invalid params error response', function (done): void {
+    // GIVEN an IssuedCurrencyClient with faked networking that will return an error response for getAccountLines
+    const invalidParamsResponse: AccountLinesResponse = {
+      result: {
+        error: 'invalidParams',
+        status: 'error',
+      },
+    }
+    const fakeErroringJsonClientResponses = new FakeJsonNetworkClientResponses(
+      invalidParamsResponse,
+    )
+    const fakeErroringJsonClient = new FakeJsonNetworkClient(
+      fakeErroringJsonClientResponses,
+    )
+    const issuedCurrencyClient = new IssuedCurrencyClient(
+      fakeSucceedingGrpcClient,
+      fakeErroringJsonClient,
+      XrplNetwork.Test,
+    )
+    // WHEN getTrustLines is called THEN an error is propagated.
+    issuedCurrencyClient.getTrustLines(testAddress).catch((error) => {
+      assert.typeOf(error, 'Error')
+      // TODO: why doesn't this work?
+      // I get: AssertionError: expected [Error: invalidParams] to deeply equal [Error: invalidParams]
+      // ( and same when I use assert.equal instead of assert.deepEqual)
+      // assert.deepEqual(
+      //   error,
+      //   new XrpError(XrpErrorType.Unknown, 'invalidParams'),
+      // )
+      done()
+    })
```

Comment:
same here

Reply:
nice, yeah this must be a relic of me copying something Keefer once did and then allowing that format to propagate.  Fixed the format for all `getTrustLines` tests.